examples/utils/example-utils/src/modelLoader/modelLoader.ts

```diff
@@ -98,7 +98,14 @@ export class ModelLoader<ModelType> implements IModelLoader<ModelType> {
     }
 
     public async loadExisting(id: string): Promise<ModelType> {
-        const container = await this.loader.resolve({ url: id });
+        const container = await this.loader.resolve({
+            url: id,
+            headers: {
+                loadMode: {
+                    opsBeforeReturn: "all",
```

Comment:
Would it be worth adding a brief comment explaining why we're enabling this?

Reply:
Done in latest